source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -751,8 +751,13 @@ def getMetricNumericValue(metricName, metricVal)
             metricValue.chomp!("k")
             metricValue = Float(metricValue) * 1000.0 ** 1
           elsif (metricValue.end_with?("m"))
+            #original value before downcase ending with M is megabyte and value ending with m is milli-byte
```

Comment:
Amol, change looks ok. But please log a future work-item to make this function all case-insensitive, so we can add support for future units easily.